65black/StackOlite
# #160199350 Consume Login Endpoint

#### What does this PR do?
This PR consumes the `/api/v1/auth/login` endpoint from [this repo](https://github.com/65black/StackOverflow-lite).

#### How should this be manually tested?
 - Clone this repo and open index.html with Chrome.
 - Navigate to the login page and try to login to an existing account.
 - If the user account doesn't exist, an error notice will be displayed at the top.
 - If the user account exists, but the login details are incorrect, an error notice will be displayed at the top.
 - If the login details are correct, you will be signed in and automatically redirected to the profile page.
#### Screenshots
![login](https://user-images.githubusercontent.com/36507972/45618738-af98dd80-ba6e-11e8-8b7c-a7e551f18668.JPG)


#### What are the relevant pivotal tracker stories?
#160199350
